h2o-r/h2o-DESCRIPTION.template

```diff
@@ -103,3 +108,5 @@ Collate:
          'zzz.R'
          'tf-idf.R'
          'rulefit.R'
+         'mli.R'
```

Comment:
Can we name this file, `explain.R` so it's more obvious what it contains?  I don't think the "MLI" acronym is universally known/used and this code is focused more on explainability vs interpretability.